### PR TITLE
feat(balance): round ~1 pound to 400g, 500g

### PR DESCRIPTION
**Pull request by scarf005**
## Purpose of change

- #2505

[metricate](https://en.wikipedia.org/wiki/Metrication) item weights, part 1.

1. many items weighted around a pound to depict them as 'somewhat heavy'.
2. it could have been weighed 500g or 400g instead.
3. eradicate trace of customary units from the game because SI units are [superior](https://old.reddit.com/r/explainlikeimfive/comments/4l5jn8/eli5_why_is_the_metric_system_seen_as_superior_to/) in many ways (easy to reason with, used by many, etc)

## Describe the solution

replaced all items weighting 450 ~ 500g to 500g, and items weighting 400 ~ 450 to 400g.

## Describe alternatives you've considered

making them all 500g to be nice and simple.

## Testing

passed automated tests in local environment.

## Additional context

~98.4% of existing countries have fully adopted metric system (192/195 countries, with the exception of Liberia, Myanmar, and the United States.)

### CODE REVIEW

File: data/json/items/chemicals_and_resources.json [hunk 1]
```diff
@@ -313,7 +313,7 @@
     "material": "powder",
     "volume": "125ml",
     "charges": 400,
-    "//2": "Previously 10 doses to a gram, so about 400 to the pound.",
+    "//2": "Previously 10 doses to a gram, so about 400g to the 500g.",
```
Comment (chaosvolt) [inline data/json/items/chemicals_and_resources.json:316]: 400 grams to the 500 grams? Just delete this comment if it's not going to be actually useful anymore.

File: data/json/items/gun/22.json [hunk 2]
```diff
@@ -118,7 +118,7 @@
     "copy-from": "ruger_lcr_38",
     "type": "GUN",
     "name": { "str_sp": "Ruger LCR .22" },
-    "weight": "420 g",
```
Comment (RobbieNeko) [inline data/json/items/gun/22.json:121]: This seems a little out of place for this PR, given that a Ruger LCR .22 is legitimately about 420 grams IRL (422.4 to be exact). Considering this PR is stated to be primarily about handling cases where something was set to be about a pound for the sake of just making it "somewhat heavy", I'd hesitate to say this really fits. 420 is already a perfectly round number.

File: data/json/items/chemicals_and_resources.json [hunk 3]
```diff
@@ -564,7 +564,7 @@
     "flags": [ "DROP_ACTION_ONLY_IF_LIQUID" ],
     "description": "Sulfuric acid: viscous, foul-smelling and extremely corrosive, particularly to organic matter and evil clowns.  Although used to prepare a number of chemicals, it is mostly known as battery acid, due to its widest commercial application.",
     "volume": "250 ml",
-    "weight": "460 g",
```
Comment (RobbieNeko) [inline data/json/items/chemicals_and_resources.json:567]: 460 is actually about the correct weight for 250 ml of sulfuric acid (the calculated weight based on density being 457.5)

**Comment by RobbieNeko** [comment]
Looks to me like a couple chemicals and firearms got caught in the crossfire when in actuality they've got pretty accurate weights that are still nice rounded numbers regardless

**Comment by VissValdyr** [comment]
While not in the 500 but rather in the 1500 range, books need some help as well. 
`
{
  "type": "BOOK",
  "name": {
    "str": "Internal Combustion Fundamentals",
    "str_pl": "copies of Internal Combustion Fundamentals"
  },
  "description": "A college-level textbook that details the operation, maintenance, and repair of internal combustion engines.",
  "bashing": 4,
  "material": [
    "paper"
  ],
  "symbol": "?",
  "looks_like": "story_book",
  "color": "blue",
  "flags": [
    "TINDER",
    "FLAMMABLE"
  ],
  "id": "book_icef",
  "copy-from": "base_book",
  "weight": "1587 g",
  "volume": "1500 ml",
  "price": "5950 cent",
  "price_postapoc": "2250 cent",
  "skill": "mechanics",
  "required_level": 4,
  "max_level": 8,
  "intelligence": 6,
  "time": "30 m",
  "fun": -1
}
`

File: data/json/items/gun/22.json (hunk 2 above)
Comment (scarf005) [reply data/json/items/gun/22.json:121]: true, tho i was thinking of #2505

Comment (RobbieNeko) [reply data/json/items/gun/22.json:121]: Fair, though I think that nearest 10 grams is already within #2505 's goal. 100% would support rounding the Cobray Streetsweeper to 420g, for example. (And magazines I have no qualms over how you adjust them, those are a lot more variable anyway)

Comment (scarf005) [reply data/json/items/gun/22.json:121]: makes sense. will revert it when i get home.

File: data/json/items/chemicals_and_resources.json (hunk 1 above)
Comment (chaosvolt) [inline data/json/items/chemicals_and_resources.json:316]:
```suggestion
```

File: data/json/items/chemicals_and_resources.json [hunk 4]
```diff
@@ -564,7 +564,7 @@
...
+    "weight": "500 g",
```
Comment (chaosvolt) [inline data/json/items/chemicals_and_resources.json:567]: Probably safe to keep chemical weights unrounded, not super important though.
```suggestion
    "weight": "460 g",
```

File: data/json/items/chemicals_and_resources.json [hunk 5]
```diff
@@ -585,7 +585,7 @@
     "flags": [ "DROP_ACTION_ONLY_IF_LIQUID" ],
     "description": "Hydrochloric acid, also known as muriatic acid.  It is a strong acid that has a distinctive smell, important and frequently used ever since its discovery.  It still has a vast multitude of uses.",
     "volume": "250 ml",
-    "weight": "460 g",
+    "weight": "500 g",
```
Comment (chaosvolt) [inline data/json/items/chemicals_and_resources.json:588]: As above
```suggestion
    "weight": "460 g",
```

File: data/json/items/gun/22.json [hunk 6]
```diff
@@ -118,7 +118,7 @@
...
+    "weight": "400 g",
```
Comment (chaosvolt) [inline data/json/items/gun/22.json:121]: As per discussion between Scarf and Robbie, reversion of gun weights since already at nearest 10 grams
```suggestion
    "weight": "420 g",
```

File: data/json/items/gun/22.json [hunk 7]
```diff
@@ -203,7 +203,7 @@
     "type": "GUN",
     "name": { "str": "Walther P22" },
     "description": "The Walther P22 is a blowback operated semi-automatic pistol.  It is about half the size of most Walthers.  It is made mostly with plastic, with the slide and key components being made of die-cast zinc alloy",
-    "weight": "450 g",
+    "weight": "500 g",
```
Comment (chaosvolt) [inline data/json/items/gun/22.json:206]:
```suggestion
    "weight": "450 g",
```

File: data/json/items/gun/32.json [hunk 8]
```diff
@@ -5,7 +5,7 @@
     "type": "GUN",
     "name": { "str_sp": "SIG Sauer P230" },
     "description": "The SIG Sauer P230 is a small, semi-automatic handgun chambered in .32 ACP.  Due to its small dimensions, it was often carried as a backup weapon.",
-    "weight": "460 g",
+    "weight": "500 g",
```
Comment (chaosvolt) [inline data/json/items/gun/32.json:8]:
```suggestion
    "weight": "460 g",
```

File: data/json/items/gun/shot.json [hunk 9]
```diff
@@ -511,7 +511,7 @@
     "type": "GUN",
     "name": { "str": "Cobray Streetsweeper" },
     "description": "Less shotgun and more comically oversized revolver, the Cobray Streetsweeper sold poorly before it was deemed a destructive device.  The cylinder is driven by a clockspring, cannot be indexed by hand, and must be ejected with an ejector rod.  Its unique design allows for all 12 shells to be fired in under 3 seconds, as demonstrated by the ATF technical branch.",
-    "weight": "419 g",
+    "weight": "400 g",
```
Comment (chaosvolt) [inline data/json/items/gun/shot.json:514]: Went with nearest 10 grams for this one instead since gun, also 420 teehee
```suggestion
    "weight": "420 g",
```

**Review by chaosvolt — CHANGES_REQUESTED** [review]

**Review by chaosvolt — APPROVED** [review]
Hopefully is okay to revert just those items mentioned, everything else looks good. :3

**Review by chaosvolt — APPROVED** [review]
